src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -49,49 +36,85 @@
         c.command('failover', 'failover_replication_link')
         c.command('force-failover', 'failover_replication_link_allow_data_loss')
 
-    with s.group('sql db data-warehouse') as c:
-        c.command('pause', 'pause_data_warehouse')
-        c.command('resume', 'resume_data_warehouse')
+    ## Data Warehouse will not be included in the first batch of GA commands
+    #with s.group('sql db data-warehouse') as c:
+    #    c.command('pause', 'pause_data_warehouse')
+    #    c.command('resume', 'resume_data_warehouse')
+
+    ## Data Warehouse will not be included in the first batch of GA commands
+    ## (list_restore_points also applies to db, but it's not very useful. It's
+    ## mainly useful for dw.)
+    #with s.group('sql db restore-point') as c:
+    #    c.command('list', 'list_restore_points')
 
-    with s.group('sql db restore-point') as c:
-        c.command('list', 'list_restore_points')
+    ## TDE will not be included in the first batch of GA commands
+    #with s.group('sql db transparent-data-encryption') as c:
+    #    c.command('create', 'create_or_update_transparent_data_encryption_configuration')
+    #    c.command('show-configuration', 'get_transparent_data_encryption_configuration')
+    #    c.command('show-activity', 'list_transparent_data_encryption_activity')
 
-    with s.group('sql db transparent-data-encryption') as c:
-        c.command('create', 'create_or_update_transparent_data_encryption_configuration')
-        c.command('show-configuration', 'get_transparent_data_encryption_configuration')
-        c.command('show-activity', 'list_transparent_data_encryption_activity')
+    ## Service tier advisor will not be included in the first batch of GA commands
+    #with s.group('sql db service-tier-advisor') as c:
+    #    c.command('list', 'list_service_tier_advisors')
+    #    c.command('show', 'get_service_tier_advisor')
 
-    with s.group('sql db service-tier-advisor') as c:
-        c.command('list', 'list_service_tier_advisors')
-        c.command('show', 'get_service_tier_advisor')
+###############################################
+#                sql elastic-pool             #
+###############################################
 
 elasticpools_ops = create_service_adapter('azure.mgmt.sql.operations.elastic_pools_operations',
                                           'ElasticPoolsOperations')
 
 with ServiceGroup(__name__, get_sql_elasticpools_operations, elasticpools_ops) as s:
-    with s.group('sql elastic-pools') as c:
-        c.command('create', 'create_or_update')
+    with s.group('sql elastic-pool') as c:
```

Comment:
I initially thought -p, although that conflicts with --admin-password -p. That might not be so bad because --admin-password is on create server, but --elastic-pool is on create/update db.

Saloni had specced -e, although that would potentially conflict with --edition -e if we wanted that alias. Also maybe not a big deal, --edition probably doesn't need an alias (since it's implied by service objective).